### ISSUE TITLE
Generate Scaffold/Model Fails: Failed to remove file

### ISSUE THREAD

**Issue by ellsong** [loco-rs/loco#714]
**Description**

I have started a new project on Windows and try to generate a new scaffold or model using a command such as:

`cargo loco generate scaffold orders state:string!`

I get the following error after some execution:

```
error: failed to remove file `D:\code\project\target\debug\project-cli.exe`

Caused by:
  Access is denied. (os error 5)
Error: Message("failed to run loco db migration. error details: `command [\"cargo\", \"loco\", \"db\", \"migrate\"] exited with code 101`")
```

**To Reproduce**

- Start a fresh project on a Windows machine
- Generate or model a scaffold

**Expected Behavior**

Successful execution

**Environment:**

Windows 11
VS Code

**Comment by jondot** [comment]
Hi, 
I'm not sure this is actually about removing a file (can't think of anywhere we need to. remove a file while doing migration)
Can you check if you have `sea-orm-cli`?

**Comment by ellsong** [comment]
> Hi, I'm not sure this is actually about removing a file (can't think of anywhere we need to. remove a file while doing migration) Can you check if you have `sea-orm-cli`?

I have both `loco-cli` and `sea-orm-cli` installed to the latest version. I am getting this error using a regular terminal as well, and when run as an administrator. This does work correctly when done via WSL.

**Comment by hrl1058** [comment]
I have same problem
But i find out
(in my case,  generate model)
1.Run cargo loco generate model xxx (will fail caused by access denied, but the migration file mxxx.rs and db table seems created normaly)
2.Run cargo loco db entities (will generate model and model/_entities normaly)
wired...

**Comment by jondot** [comment]
were still resolving this one. it is due to windows locking binary files as they run.
we're testing a solution

**Comment by jondot** [comment]
now solved in 0.8.1

this specific test case shows that it works on windows: https://github.com/loco-rs/loco/actions/runs/10663590236/job/29553083714

thanks for catching this!
